internal/backend/grpc/management.go

```diff
@@ -72,6 +74,29 @@ type JWTSigningKeyProvider interface {
 	SigningKey(ctx context.Context) (op.SigningKey, error)
 }
 
+func newManagementServer(
+	omniState state.State,
+	jwtSigningKeyProvider JWTSigningKeyProvider,
+	logHandler *siderolink.LogHandler,
+	logger *zap.Logger,
+	dnsService *dns.Service,
+	imageFactoryClient *imagefactory.Client,
+	auditor AuditLogger,
+	omniconfigDest string,
+) *managementServer {
+	return &managementServer{
+		omniState:             omniState,
+		jwtSigningKeyProvider: jwtSigningKeyProvider,
+		logHandler:            logHandler,
+		logger:                logger,
+		dnsService:            dnsService,
+		imageFactoryClient:    imageFactoryClient,
+		auditor:               auditor,
+		omniconfigDest:        omniconfigDest,
+		jsonSchemaCache:       expirable.NewLRU[string, *jsonschema.Schema](64, nil, 0),
```

Comment:
Probably not a big deal, but this size (64) is for the number of items I believe, not caring about the size of a single entry. That's why I suggested something like a [cost-based eviction](https://github.com/hypermodeinc/ristretto), each item's cost would be the size of their raw schema bytes. I don't know if there's a way to do it with this LRU.

Reply:
I'd say just removing free-form JSON schema removes the whole problem, and probably cache isn't actually needed?

We validate a specific provider schema

Reply:
The component for the JsonForms is generic. I'd rather validate it on the frontend, but as that's not possible anymore I moved it to backend.

Of course it's doable, we can lock it to the providers only. But don't really want to do that.

Reply:
We could implement a generic resource viewer and editor based on this JsonForm component in the future.
If we lock it to the provider schema, it won't be possible.